client/components/Nav.jsx

```diff
@@ -487,7 +487,7 @@ class Nav extends React.PureComponent {
               }}
             >
               <span className="nav__item-header">Help</span>
-              <InlineSVG className="nav__item-header-triangle" src={triangleUrl} />
+              <InlineSVG className="nav__item-header-triangle" src={triangleUrl} title="Help" />
```

Comment:
Hey @ashu8912 , I don't think your fix served its purpose. I am able to see incorrect labels inside the dropdown lists:-

![screenshot](https://user-images.githubusercontent.com/30630904/75064308-b3a31780-550c-11ea-99c3-4448b726f9ab.png)